README.md

```diff
@@ -1,14 +1,14 @@
-# NS Theme Check
+# Theme Sniffer
 
-Theme Check plugin using sniffs. `WordPress-Theme` standard is used from [WPTRT/WordPress-Coding-Standards](https://github.com/WPTRT/WordPress-Coding-Standards).
+Theme Sniffer plugin using sniffs. `WordPress-Theme` standard is used from [WPTRT/WordPress-Coding-Standards](https://github.com/WPTRT/WordPress-Coding-Standards).
 
 ![Screenshot](screenshot.png?raw=true)
 
 ## Installing
 
 ### Option 1: Easy
 
-* Download [zip file](https://github.com/ernilambar/ns-theme-check/releases/download/0.1.4/ns-theme-check.0.1.4.zip). [Note: Please use this distribution plugin zip. GitHub provided zip will not work.]
+* Download [zip file](https://github.com/ernilambar/theme-sniffer/releases/download/0.1.4/theme-sniffer.0.1.4.zip). [Note: Please use this distribution plugin zip. GitHub provided zip will not work.]
```

Comment:
Repo has been moved. And also repo name will be changed. It will be probably https://github.com/WPTRT/theme-sniffer Please update accordingly